README.md

```diff
@@ -130,5 +130,4 @@ guide](https://docs.bazel.build/versions/3.2.0/install.html)).
 Bazel will pull pypi libraries automatically, but you need to have pip3
 installed.
 
-Then you can feed data into the program by writing it as a string to standard
-input, and execute `bazel-bin/main/train_classifier_g`.
+To run it, you need to put your `feed_data.R` file in `main/`, then build with `bazel build //main:feed_data` in the root directory, which is where `WORKSPACE` file locates, run with `bazel-bin/main/feed_data`
```

Comment:
Since feed_data is already present, can't this whole comment be reduced to:

To run it, execute `bazel run //main:feed_data`.

Reply:
ok. Changed.